components/data/data-services/org.wso2.micro.integrator.dataservices.core/src/test/java/org/wso2/micro/integrator/dataservices/core/test/util/TestUtils.java

```diff
@@ -71,6 +72,11 @@
 	public static final String ORDER_DETAILS_NESTED_XSD_PATH = "./src/test/resources/xsd/order_details_nested.xsd";
 	
 	public static final String OFFICES_XSD_PATH = "./src/test/resources/xsd/offices.xsd";
+	public static final String OFFICES_XSD_PATH2 = "./src/test/resources/xsd/offices.xsd";
+
+	public TestUtils(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/app/sdkottegoda/micro-integrator/pullRequest?prid=4580919)